README.md

```diff
@@ -115,6 +115,7 @@ Various inputs are defined in [`action.yml`](action.yml) to let you configure th
 | `configuration-path` | The path to the label configuration file | `.github/labeler.yml` |
 | `sync-labels` | Whether or not to remove labels when matching files are reverted or no longer changed by the PR | `false` |
 | `dot` | Whether or not to auto-include paths starting with dot (e.g. `.github`) | `false` |
+| `pr-number` | The PR number of a PR that to update, rather than detecting from the workflow context | N/A |
```

Comment:
We should add there information that it can accept list of pr numbers. The wording feels slightly off. Also that would be great if you could add the example of using this input with multiple pr numbers.